packages/client/hmi-client/src/components/workflow/ops/model-config/tera-model-config-drilldown.vue

```diff
@@ -7,6 +7,18 @@
 		hide-dropdown
 	>
 		<template #sidebar>
+			<tera-slider-panel
+				v-if="pdfData"
+				v-model:is-open="isDocViewerOpen"
+				header="Document Viewer"
+				content-width="700px"
+			>
+				<template #content>
+					<tera-drilldown-section :is-loading="isFetchingPDF">
+						<tera-pdf-panel :pdfs="pdfData" />
```

Comment:
should we hide the pdf panel when there is no attached document to the model config node?
<img width="1473" alt="Screenshot 2024-10-17 at 10 48 18 AM" src="https://github.com/user-attachments/assets/41abff22-eeba-4fb0-83f7-931f0a0d512c">